src/Cooked/Attack/DoubleSat.hs

```diff
@@ -206,22 +137,10 @@ doubleSatAttack optic extra attacker mode = do
         >> mapM_ addOutputTweak outs
         >> mapM_ addMintTweak (txSkelMintsToList mints)
 
-    -- Join a list of 'DoubleSatDelta's into one 'DoubleSatDelta' that specifies
-    -- eveything that is contained in the input.
-    joinDeltas :: [DoubleSatDelta] -> DoubleSatDelta
-    joinDeltas = mconcat
-
-    sameDeltas :: DoubleSatDelta -> DoubleSatDelta -> Bool
-    sameDeltas (i, o, m) (i', o', m') = i == i' && sameMultiSet o o' && m == m'
-      where
-        sameMultiSet :: Eq a => [a] -> [a] -> Bool
-        sameMultiSet [] [] = True
-        sameMultiSet (x : xs) ys = sameMultiSet xs (delete x ys)
-        sameMultiSet _ _ = False
-
-    allCombinations :: [[x]] -> [[x]]
-    allCombinations (l : ls) = let cs = allCombinations ls in concatMap (\x -> (x :) <$> cs) l
-    allCombinations [] = [[]]
+-- Join a list of 'DoubleSatDelta's into one 'DoubleSatDelta' that specifies
+-- eveything that is contained in the input.
+joinDoubleSatDeltas :: [DoubleSatDelta] -> DoubleSatDelta
+joinDoubleSatDeltas = mconcat
```

Comment:
This is only meant as an auxiliary function for the doubleSat attack, right? Or would this have a use elsewhere? Should we export it?